org.eclipse.scout.rt.server/src/main/java/org/eclipse/scout/rt/server/context/ServerRunContextProducer.java

```diff
@@ -43,6 +44,7 @@ public class ServerRunContextProducer extends RunContextProducer {
   public ServerRunContext produce(final Subject subject) {
     final ServerRunContext serverRunContext = ServerRunContexts.copyCurrent(true)
         .withSubject(subject)
+        .withThreadLocal(Users.CURRENT, BEANS.get(IAccessControlService.class).getUserId(subject))
```

Comment:
In client the user is added in org.eclipse.scout.rt.client.context.ClientRunContext#interceptCallableChain
Check client vs. server and use same location to add user thread local.

Preferrable in XyRunContext#interceptCallableChain
And maybe remove then in org.eclipse.scout.rt.server.commons.context.HttpRunContextProducer#produce(jakarta.servlet.http.HttpServletRequest, jakarta.servlet.http.HttpServletResponse, org.eclipse.scout.rt.platform.context.RunContext)